CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.0] - 2020-09-09
+
+Props: GaryJones, jrfnl, rebeccahum.
+
+### Added
+- [#494](https://github.com/Automattic/VIP-Coding-Standards/pull/494): `.gitattributes` file.
+- [#495](https://github.com/Automattic/VIP-Coding-Standards/pull/495): `CODEOWNERS` file.
+- [#450](https://github.com/Automattic/VIP-Coding-Standards/pull/450): [VariableAnalysis](https://github.com/sirbrillig/phpcs-variable-analysis/) package.
```

Comment:
Should a "Deprecated" header be added with an entry that the old VIPCS version of this sniff is now deprecated and will be removed in the next major ?

Reply:
Added.